src/components/layouts/register/layout.styles.tsx

```diff
@@ -0,0 +1,79 @@
+import { gridBreakpoint } from "@/shared/breakpoints";
+import styled from "styled-components";
+
+export const StylesWrapper = styled.div<{}>`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  text-align: center;
+
+  .image-side {
+    position: relative;
+    width: 100vw;
+
+    div {
+      position: absolute;
+      top: 50%;
+      left: 50%;
+      text-align: center;
+      transform: translate(-50%, -50%);
+      color: white;
+      text-shadow: 1px 1px 4px #000000;
+
+      h3 {
+        font-size: 30px;
+      }
+
+      h1 {
+        font-size: 48px;
+      }
+    }
+  }
+
+  .form-wrapper > form {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    text-align: center;
+    h1 {
+      display: none;
+    }
+    .input-wrapper {
+      margin: 2rem 0 0 0;
+    }
+
+    .error-message {
+      color: red;
```

Comment:
🦆 This use of color might be needed somewhere at some other point. In general it is a good practice to add colors to your global palette file, where you can keep track of which ones did you use. Later, you can reference this colors using a variable for instance.

```
--error-color: red;

color: var(--error-color);
```

This are good practice that will help you to keep a consistent design through your application.

Reply:
Note, that this can also be done for a pack of styles! You can declare a css``, block from styled components somewhere, and then use it in your application. Same as the `raleway` bundle for example.

Reply:
🥔 For this one, I've created a file to declare a css`` block to create the error-message style and I've implemented them for the register and login form